### PR TITLE
Slides: GRASS GIS code manual in Markdown

### PR DESCRIPTION
**Pull request by neteler**
Two slides added (might be reduced to one)

I can also redo the screenshot(s).

### CODE REVIEW

**Comment by neteler** [comment]
Please don't merge yet, I'll redo the screenshots.

**Review by pesekon2 — APPROVED** [review]

File: foss4g2023.html [hunk 1]
```diff
@@ -807,6 +807,25 @@ <h2>Spatial Query of Projections</h2>
         by Huidae Cho</p>
 </section>
 
+<section>
+    <h1>Conversion of the manual to Markdown/mdocs</h1>
```
Comment (veroandreo) [inline foss4g2023.html:811]:
```suggestion
    <h1>Conversion of manuals to Markdown/mdocs</h1>
```

File: foss4g2023.html [hunk 2]
```diff
@@ -807,6 +807,25 @@ <h2>Spatial Query of Projections</h2>
...
+    <br><br>
+    Goal: easier to maintain and more friendly to collect user contributions
+    <br>
+    <img src="img/grass_manual_readthedocs_1.png" alt="GRASS GIS code manual in Markdown" width="45%">
+    <p>by Markus Neteler</p>
+    <br><b style='color:red;'>TODO: move to better section</b>
+</section>
+
+<section>
+    <h1>Conversion of the manual to Markdown/mdocs</h1>
```
Comment (veroandreo) [inline foss4g2023.html:822]:
```suggestion
    <h1>Conversion of manuals to Markdown/mdocs</h1>
```

**Review by veroandreo — COMMENTED** [review]
The conversion looks really nice! Minor suggested fix for slide titles

**Comment by landam** [comment]
Thanks a lot for contribution! @neteler Can we merge?

**Comment by neteler** [comment]
Ready to merge!

**Comment by landam** [comment]
Thanks for contribution! @neteler

**Pull request merged**